pkg/pillar/cmd/zedrouter/pubsubhandlers.go

```diff
@@ -146,6 +185,28 @@ func (z *zedrouter) handleDNSDelete(ctxArg interface{}, key string,
 	z.log.Functionf("handleDNSDelete done for %s", key)
 }
 
+func (z *zedrouter) handleWwanMetricsCreate(ctxArg interface{}, key string,
```

Comment:
How do there Wwan metrics relate to the multiple ports work?

Reply:
This was requested by a customer: https://github.com/lf-edge/eve-api/blob/main/proto/config/netinst.proto#L121-L125
With multiple modems, portprober will prefer modem with better network signal.
Zedrouter needs to subscribe to wwan metrics with signal strength info and pass that to the portprober.

Reply:
Ah - I just didn't see it as a top-level description but the "cost etc" is referencing signal strength. Got it.